sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/surveillance/DiseaseStatisticsComponent.java

```diff
@@ -58,8 +58,11 @@ public DiseaseStatisticsComponent(DashboardDataProvider dashboardDataProvider) {
 		caseStatisticsComponent = new CaseStatisticsComponent();
 		diseaseSummaryComponent = new DiseaseSummaryComponent();
 		eventStatisticsComponent = new EventStatisticsComponent();
-		testResultsStatisticsComponent =
-			new TestResultsStatisticsComponent(Captions.dashboardNewTestResults, Descriptions.descDashboardNewTestResults, null, false);
+		testResultsStatisticsComponent = new TestResultsStatisticsComponent(
```

Comment:
testResultsStatisticsComponent and TestResultsStatisticsComponent class should be also renamed to finalLaboratoryResultStatisticsComponent (or finalLaboratoryResultComponent )
 and FinalLaboratoryResultStatisticsComponent (or FinalLaboratoryResultComponent)